packages/block-library/src/post-date/block.json

```diff
@@ -23,7 +23,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
This block has a "Link to post" setting. Should we add `link` as one of the defaults?